doc/book/intro.md

```diff
@@ -1,6 +1,6 @@
 # Introduction
 
-`laminas-permissions-rbac` provides a lightweight [Role-Based Access Control](https://it.wikipedia.org/wiki/Role-based_access_control)
+`laminas-permissions-rbac` provides a lightweight [Role-Based Access Control](https://en.wikipedia.org/wiki/Role-based_access_control)
```

Comment:
You can remove `en` or replaced with `www`, the result is the same.

Reply:
@froschdesign I think English is fine, as our documentation is in English only.

Reply:
It will redirect to the English version.